faker/tests/requirements.txt

```diff
@@ -0,0 +1 @@
+mock==1.0.1
```

Comment:
Releases will have to be coordinated with the original author. I can bump up version numbers but I can't upload to PyPI. Would that work for you?

I could ping the author about uploaidng a release, but he's currently busy working on his Master's Thesis and I'd like let him focus on that :)